tests/utils.py

```diff
@@ -75,6 +74,9 @@ def __init__(
         self.host = str(args.host or 'localhost')
         self.port = int(args.port)
 
+        if max_start_wait_s:
+            self.MAX_START_WAIT_S = max_start_wait_s
```

Comment:
We should not update the class-wide constant. Instead, keep a separate instance attribute.

Reply:
That's what this does. Updating the class-wide constant would be `RemoteOpenAIServer.MAX_START_WAIT_S = max_start_wait_s`. Using `self.` instead of `RemoteOpenAIServer.` means it's only modified for the instance. A trivial example:
<img width="677" alt="Screenshot 2024-08-16 at 8 36 52 AM" src="https://github.com/user-attachments/assets/39b091a4-f853-495b-9bb6-fcbe5e764161">

Reply:
To avoid confusion, I think it would be clearer to use lower case for the instance attribute.

Reply:
I agree, all caps signifies global variables generally

Reply:
ok - I am not the one who implemented it originally so I didn't mess with it, but I'm happy to refactor it.

Reply:
it looks like the only place it's used is internal to the class, so there's no reason it has to be a separate instance and class variable - I refactored it to make it only an instance variable with a default value.